TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/fragments/GlobeFragment.java

```diff
@@ -0,0 +1,91 @@
+package com.fbu.thefoodienetwork.fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.fbu.thefoodienetwork.R;
+import com.fbu.thefoodienetwork.adapters.ReviewAdapter;
+import com.fbu.thefoodienetwork.databinding.FragmentGlobeBinding;
+import com.fbu.thefoodienetwork.models.ParseReview;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GlobeFragment extends Fragment {
+    private static final String TAG = "GlobeFragment";
+    FragmentGlobeBinding binding;
+    protected ReviewAdapter reviewAdapter;
+    protected List<ParseReview> allReviews;
+    private LinearLayoutManager layoutManager;
+    private RecyclerView reviewRecyclerView;
+
+    public GlobeFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
```

Comment:
I Love that you add a lot of comments, however, it seems that some of these may just be due to copying and pasting old code.  This is ok, if it actually helps you, but I have a feeling that you know what `onCreateView` does at this point, so the comment might not actually be providing any value.